spec/models/export/azure_upload_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Export::AzureUpload do
+  let(:files_map) { { 'file1.csv' => file_1, 'file2.json' => file_2 } }
+  let(:file_1) { File.open(Rails.root.join('spec', 'fixtures', 'users.csv')) }
+  let(:file_2) { File.open(Rails.root.join('spec', 'fixtures', 'bank_holidays.json')) }
+
+  subject(:azure_upload) { Export::AzureUpload.new(files_map) }
+
+  let(:blob_client) { spy('blob_client') }
+
+  around do |example|
+    ClimateControl.modify AWS_S3_EXPORT_BUCKET: 'test-bucket' do
+      example.run
+    end
+  end
+
+  describe '#perform' do
+    it 'uploads each file to the configured S3 bucket' do
```

Comment:
`it 'uploads each file to the configured Azure container' do`

Reply:
🤦‍♂️